pages/src/source/workers/chickenfarmer.md

```diff
@@ -1,41 +1,27 @@
 ---
-title: Chicken Farmer
-layout: default
+type: worker
+worker: chickenfarmer
+layout: worker
 ---
-# Chicken Farmer
+{% capture content %}
+The Carpenter is a part of your colony's production line. The Carpenter can learn 3x3 recipes made of at least 75% wood that do not include ingots, stone, redstone (or produce a redstone signal), or string. The Carpenter will also craft a few other items, including [cactus planks](../../source/items/cactusplanks), [shingles](../../source/items/shingles), [timber frames](../../source/items/timberframes), [compost barrels](../../source/items/compostbarrel), and [racks](../../source/items/rack). The Carpenter will only make these items when they have been taught the recipes, receive a request for an item, and have the needed materials. 
```

Comment:
Are you sure about this description? I'm a bit skeptical about what happened here.

Reply:
I fixed that one, it was the only content page which was wrong